streaming/base/format/base/writer.py

```diff
@@ -323,15 +300,17 @@ def __init__(self,
                  hashes: Optional[List[str]] = None,
                  size_limit: Optional[int] = 1 << 26,
                  extra_bytes_per_shard: int = 0,
-                 extra_bytes_per_sample: int = 0) -> None:
+                 extra_bytes_per_sample: int = 0,
+                 **kwargs: Any) -> None:
```

Comment:
Dict[str, Any]

Reply:
See my above response for the same comment.